scripts/install-cni.sh

```diff
@@ -25,10 +25,25 @@ if [ -z "${pod_cidr:-}" ]; then
 fi
 
 if [ -w /host/etc/cni/net.d ]; then
-  cni_spec=$(echo ${CNI_SPEC_TEMPLATE:-} | sed -e "s#podCidr#${pod_cidr:-}#g")
+  cni_spec=$(echo ${CNI_SPEC_TEMPLATE:-} | sed -e "s#@podCidr#${pod_cidr:-}#g")
+
+  if [ "$ENABLE_PRIVATE_IPV6_ACCESS" = true ]; then
+    node_ipv6_addr=$(curl -s -k --fail "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/ipv6s" -H "Metadata-Flavor: Google" | jq '.[0]') ||:
```

Comment:
I just tried the following commands, it works for me
"http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/?recursive=true" -H "Metadata-Flavor: Google" | jq '.ipv6s[0]'

BTW, what do -k and --fail mean for curl?